lambdas/update_service_list/update_service_list.py

```diff
@@ -14,34 +14,67 @@
 
 import boto3
 
-from ecs_utils import get_cluster_arns, get_service_arns, describe_service
+from ecs_utils import get_cluster_arns, get_service_arns, describe_service, describe_cluster
 
```

Comment:
style nit: two blank lines between adjacent functions/functions and imports in Python.